File: test/golden/java/common/src/main/java/com/example/testmod/TestmodMod.java
package com.example.testmod;

import org.slf4j.Logger;
import org.slf4j.LoggerFactory;

public class TestmodMod {
    public static final String MOD_ID = "testmod";
    public static final Logger LOGGER = LoggerFactory.getLogger(MOD_ID);

    public static void init() {
        LOGGER.info("Initializing Test Mod");
    }
}

File: test/golden/kotlin/common/src/main/kotlin/com/example/testmod/TestmodMod.kt
package com.example.testmod

import org.slf4j.LoggerFactory

object TestmodMod {
    const val MOD_ID = "testmod"
    val LOGGER = LoggerFactory.getLogger(MOD_ID)

    fun init() {
        LOGGER.info("Initializing Test Mod")
    }
}
